packages/squiggle-lang/src/rescript/Reducer/Reducer_Expression/Reducer_Expression.res

```diff
@@ -118,7 +96,11 @@ module BackCompatible = {
 
   let evaluate = (expression: t): result<InternalExpressionValue.t, errorValue> => {
     let accessors = ProjectAccessorsT.identityAccessors
-    expression->reduceExpressionInProject(accessors.stdLib, accessors)
+    try {
+      expression->reduceExpressionInProject(accessors.stdLib, accessors)->Ok
```

Comment:
This `try` strategy seems awkward with functional code to me. I'll let @umuro decide here, I believe it might have been discussed on Slack.